aries-site/src/examples/templates/code-blocks/DarkCodeTheme.js

```diff
@@ -0,0 +1,125 @@
+export const DarkCodeTheme = {
+    'code[class*="language-"]': {
+        'color': '#f8f8f8',
+        'background': 'none',
+        'textAlign': 'left',
+        'whiteSpace': 'pre',
+        'wordSpacing': 'normal',
+        'wordBreak': 'normal',
+        'wordWrap': 'normal',
+        'lineHeight': '1.5',
+        'MozTabSize': '4',
+        'OTabSize': '4',
+        'tabSize': '4',
+        'WebkitHyphens': 'none',
+        'MozHyphens': 'none',
+        'msHyphens': 'none',
+        'hyphens': 'none',
+    },
+    'pre[class*="language-"]': {
+        'color': '#f8f8f8',
```

Comment:
Wondering what the best approach is for folks to consume these themes...

Would is make sense to make these a part of `grommet-theme-hpe`?

Reply:
Instead of coding the hex values here, could we reference these colors from grommet-theme-hpe?

Reply:
How did we arrive at this value? I don't believe this is available in the HPE theme? Are we okay with that?

Reply:
I didn't initially include this in the grommet-theme-hpe since the code blocks are not a part of grommet and to use them you need an external library. I thought including them in the theme might cause confusion and people would assume code blocks are part of grommet. I am open to thoughts on this though.

Similar reasoning to above. I didn't directly reference the colors from the grommet-hpe-theme because I wasn't sure if the code block styling should be able to stand alone. Again, open to thoughts on this.

You're right '#f8f8f8' is not part of the hpe theme. I accidentally grabbed that value from the grommet theme. I will update it to an hpe theme value.

Reply:
Is there a way to include it inside the grommet-theme-hpe package but not embed it inside the current theme object. For example: `import { code-theme } from 'grommet-theme-hpe'`. This would allow tree shaking to omit it but not require a separate import.

Reply:
👆 Exporting separate objects was along the lines I was thinking.

Reply:
Created a [PR on grommet-site](https://github.com/grommet/grommet-theme-hpe/pull/214)